src/wazuh_modules/syscollector/tests/sysCollectorFlatbuffers/syscollectorFb_test.cpp

```diff
@@ -0,0 +1,322 @@
+/*
+ * Wazuh SyscollectorImp
+ * Copyright (C) 2015, Wazuh Inc.
+ * August 1, 2023.
+ *
+ * This program is free software; you can redistribute it
+ * and/or modify it under the terms of the GNU General Public
+ * License (version 2) as published by the FSF - Free Software
+ * Foundation.
+ */
+
+#include "syscollectorFb_test.h"
+
+constexpr auto SYNC_HMSG_TEST_PATH {"sync_hmsg.mom"};
+constexpr auto SYNC_PMSG_TEST_PATH {"sync_pmsg.mom"};
+
+constexpr auto COMPONENT {"syscollector"};
+constexpr auto TYPE {"state"};
+
+#define GTEST_COUT std::cerr << "[          ] [ INFO ]"
+
+
+const std::string syscollector_message {SCHEMA_ROOT_PATH "syscollector_synchronization.fbs"};
+const char* include_directories[] = { SCHEMA_ROOT_PATH, nullptr };
+
+void SyscollectorFbTest::SetUp() {};
+
+void SyscollectorFbTest::TearDown() {};
+
+TEST(SyscollectorFbTest, JSONParsePackageUnix)
+{
+
+    const std::string alert_json =
+        "{\n  agent_info: {\n    agent_id: \"001\",\n    node_name: \"node01\"\n  },\n  data_type: \"syscollector_packages\",\n  data: {\n    attributes_type: \"syscollector_pacakges\",\n    attributes: {\n      architecture: \"amd64\",\n      checksum: \"409378153d05da4d49900316be982e575cb2586b\",\n      description: \"GNU C++ compiler for MinGW-w64 targeting Win64\",\n      format: \"deb\",\n      groups: \"devel\",\n      item_id: \"65a25b9b9fe7cb173aa5cc36dc437d9875af8a8e\",\n      name: \"g++-mingw-w64-x86-64\",\n      priority: \"optional\",\n      scan_time: \"0000/00/00 00:00:00\",\n      size: 155993,\n      source: \"gcc-mingw-w64 (22~exp1ubuntu4)\",\n      vendor: \"Stephen Kitt <skitt@debian.org>\",\n      version: \"9.3.0-7ubuntu1+22~exp1ubuntu4\"\n    },\n    index: \"65a25b9b9fe7cb173aa5cc36dc437d9875af8a8e\",\n    timestamp: \"\"\n  }\n}\n";
```

Comment:
The typo "syscollector_pacakges" doesn't generate a parsing error.
Is it because the attribute is a table and not an union?